lib/net.c

```diff
@@ -54,6 +52,18 @@ uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN] = {
 	0x6a, 0x42, 0xb7, 0x3b, 0xbe, 0xac, 0x01, 0xfa,
 };
 
+/* Function to randomize the default RSS key */
+static void
+randomize_rss_key(void)
+{
+	srand(time(NULL));
+
+	for (int i=0; i<GATEKEEPER_RSS_KEY_LEN; i++) {
```

Comment:
1. Style: add a space before `=`, and another space after it. Do the same to `<`.

2. Style: Declare `i` at the beginning of the block, in this case, the function itself.